src/main/resources/scripts/sequence-files/purge_sequence_files.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python
+import argparse
+import mysql.connector
+import os
+
+def remove(path, purge):
+    if purge:
+        try:
+            if os.path.isdir(path):
```

Comment:
Can you add a check that the path exists first? When it tries to delete a directory/path that is already gone (for example, removed via admin panel in other branch) it only says `Unable to delete`. This might confuse someone why some files are not deleting, when they didn't exist in the first place.
![image](https://user-images.githubusercontent.com/16961365/222800568-bd267ba1-f19d-481f-b0d8-d5e6a8b97209.png)

Reply:
Please see 2dd698774e39cd05609d0e3f69ee472c31be573b.